NebulaClient/PacketProcessors/Logistics/ILSAddStationComponentResponseProcessor.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NebulaModel.Attributes;
+using NebulaModel.Networking;
+using NebulaModel.Packets.Logistics;
+using NebulaModel.Packets.Processors;
+using NebulaModel.DataStructures;
+using NebulaWorld;
+using NebulaWorld.Logistics;
+using UnityEngine;
+
+namespace NebulaClient.PacketProcessors.Logistics
+{
+    [RegisterPacketProcessor]
+    public class ILSAddStationComponentResponseProcessor: IPacketProcessor<ILSAddStationComponentResponse>
+    {
+        public void ProcessPacket(ILSAddStationComponentResponse packet, NebulaConnection conn)
+        {
+            for(int i = 0; i < ILSShipManager.AddStationComponentQueue[packet.planetId].Count; i++)
+            {
+                if(ILSShipManager.AddStationComponentQueue[packet.planetId][i].shipDockPos == DataStructureExtensions.ToVector3(packet.shipDockPos))
+                {
+                    LocalPlayer.PatchLocks["GalacticTransport"] = true;
```

Comment:
There is a toggleswitch abstraction now that can be used for this.

Reply:
I *think* this is not the usecase here. What i need to do is set a boolean that i can check for in my Prefix patches to determine if they should run the patch code or the vanilla code in order to avoid endless loops.